pets/mysql/galera/mysql-galera.yaml

```diff
@@ -0,0 +1,120 @@
+# A headless service to create DNS records
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
+  name: galera
+  labels:
+    app: mysql
+spec:
+  ports:
+  - port: 3306
+    name: mysql
+  # *.galear.default.svc.cluster.local
+  clusterIP: None
+  selector:
+    app: mysql
+---
+apiVersion: apps/v1alpha1
+kind: PetSet
+metadata:
+  name: mysql
+spec:
+  serviceName: "galera"
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: mysql
+      annotations:
+        pod.alpha.kubernetes.io/initialized: "true"
+        pod.alpha.kubernetes.io/init-containers: '[
+            {
+                "name": "install",
+                "image": "gcr.io/google_containers/galera-install:0.1",
+                "imagePullPolicy": "Always",
+                "args": ["--work-dir=/work-dir"],
+                "volumeMounts": [
+                    {
+                        "name": "workdir",
+                        "mountPath": "/work-dir"
+                    },
+                    {
+                        "name": "config",
+                        "mountPath": "/etc/mysql"
+                    }
+                ]
+            },
+            {
+                "name": "bootstrap",
+                "image": "debian:jessie",
+                "command": ["/work-dir/peer-finder"],
+                "args": ["-on-start=\"/work-dir/on-start.sh\"", "-service=galera"],
+                "env": [
+                  {
+                      "name": "POD_NAMESPACE",
+                      "valueFrom": {
+                          "fieldRef": {
+                              "apiVersion": "v1",
+                              "fieldPath": "metadata.namespace"
+                          }
+                      }
+                   }
+                ],
+                "volumeMounts": [
+                    {
+                        "name": "workdir",
+                        "mountPath": "/work-dir"
+                    },
+                    {
+                        "name": "config",
+                        "mountPath": "/etc/mysql"
+                    }
+                ]
+            }
+        ]'
+    spec:
+      containers:
+      - name: mysql
+        image: erkules/galera:basic
```

Comment:
That's just lifted from the official galera page: http://galeracluster.com/2015/05/getting-started-galera-with-docker-part-1/ I want to prove that I'm not using a custom image with its own entrypoing that watches etcd etc (because I don't have faith in such a solution).

To your point, I should repush this image to gcr.io for the e2e. But I'll leave this image as is for the mentioned reason? (the e2es don't use this yaml)